app/models/corporate_information_page.rb

```diff
@@ -175,8 +181,7 @@ def base_path
     return if owning_organisation.blank?
 
     url = owning_organisation.base_path + "/about/#{slug}"
-    url.gsub!("/about/about", "/about") if organisation.present?
-    url
+    url.gsub("/about/about", "/about")
```

Comment:
I wonder if this entire method could now be cleaned up, to remove the confusing gsub?

E.g.

```
    return if owning_organisation.blank?

    url = if slug == "about"
                owning_organisation.base_path + "/about"
              else
                owning_organisation.base_path + "/about/#{slug}"
              end
```

Reply:
Done 👍